pkg/cmd/admin/router/router.go

```diff
@@ -287,6 +287,7 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 							Spec: kapi.PodSpec{
 								ServiceAccount: cfg.ServiceAccount,
 								NodeSelector:   nodeSelector,
+								HostNetwork: true,
```

Comment:
It's looking more and more like host network is the right option.  The question comes from what we do to make sure it doesn't break other things (like the stat port being outwardly visible).  We may need to understand how host network abstractions fit with our model.

> On Jun 18, 2015, at 12:29 AM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In pkg/cmd/admin/router/router.go:
> 
> > @@ -287,6 +287,7 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
> >  							Spec: kapi.PodSpec{
> >  								ServiceAccount: cfg.ServiceAccount,
> >  								NodeSelector:   nodeSelector,
> > +								HostNetwork: true,
> if we do this for real, this should be an option, not hard-coded, not sure what the default should be
> 
> —
> Reply to this email directly or view it on GitHub.
>